mkosi/qemu.py

```diff
@@ -193,13 +193,17 @@ def start_swtpm() -> Iterator[Path]:
 
             cmdline += ["--ctrl", f"type=unixio,fd={sock.fileno()}"]
 
-            proc = spawn(cmdline, user=INVOKING_USER.uid, group=INVOKING_USER.gid, pass_fds=(sock.fileno(),))
-
-            try:
-                yield path
-            finally:
-                proc.terminate()
-                proc.wait()
+            with spawn(\
+                cmdline,\
+                user=INVOKING_USER.uid,\
+                group=INVOKING_USER.gid,\
+                pass_fds=(sock.fileno(),)\
```

Comment:
The line continuations in here are actually not needed (I tested this as far back as 3.5), since they are *in* the function call - where Python needs the line continuations (at least on our lower bound) is when combining multiple context managers.

Reply:
Ahh I wasn't sure, will address all of these in a separate PR